docs/reference/ingest/processors/circle.asciidoc

```diff
@@ -0,0 +1,166 @@
+[role="xpack"]
+[testenv="basic"]
+[[ingest-circle-processor]]
+=== Circle Processor
+Converts circle definitions of shapes to regular polygons which approximate them.
+
+[[circle-processor-options]]
+.Circle Processor Options
```

Comment:
probably worth adding some images to help give a visual definition to `error_distance_in_meters` and show an approximate circle-polygon, what do you think?

Reply:
added to the docs! http://elasticsearch_43851.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/ingest-circle-processor.html

Reply:
looks great!